libraries/standard/json/cbmc/sources/skipGeneric.c

```diff
@@ -0,0 +1,33 @@
+#include "skipGeneric.h"
+
+/**
+ * See skipGeneric.h for docs
+ *
+ * Advance buffer index beyond some minimum value.
+ */
+static bool_ skipGeneric( const char * buf,
```

Comment:
This file concerns me, because we're literally doing nothing involving the buffer. So even if there was a memory-safety problem, we wouldn't necessarily catch it.

Reply:
This servers to mimic the index manipulations for the replacement functions.  For every replacement function there is a proof which exercises the original function directly including the memory accesses.

Reply:
I assume there is a relationship between `buf`, `*start`, `min` and `max`.  For e.g. could you do something like
`assert(__CPROVER_r_ok(buf + *start, max))`?

Reply:
added